src/main/java/lotto/domain/LottoRank.java

```diff
@@ -0,0 +1,44 @@
+package lotto.domain;
+
+public enum LottoRank {
+    FAIL(2, 0L, false),
+    THREE(3, 5000L, false),
```

Comment:
언더바를 이용하면 가독성을 높일 수 있어요!
```suggestion
    THREE(3, 5_000L, false),
```

Reply:
배워갑니다!!